src/test/java/unit/test/VirtualItemTest.java

```diff
@@ -0,0 +1,31 @@
+package unit.test;
+
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.Test;
+import shop.VirtualItem;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+class VirtualItemTest {
```

Comment:
RealItem and VirtualItem classes were designed as simple classes without any business logic and it was a trap in task description. There is no needs to write tests for these classes at all, the only method for which it can be done - toString(). Do not write tests for simple operations (get/set). Please remove test and add for toString() (not obligatory)